### PR TITLE
Update ceilometer compute agent template

### PR DESCRIPTION
**Pull request by vyzigold**

### CODE REVIEW

File: templates/ceilometercompute/config/ceilometer_agent_compute.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "image": "{{ .ceilometer_compute_image }}",
-    "user": "root",
+    "user": "ceilometer",
```
Comment (jlarriba) [inline templates/ceilometercompute/config/ceilometer_agent_compute.json:3]: Does the user Ceilometer have enough permissions to connect to libvirt socket?

Comment (vyzigold) [reply templates/ceilometercompute/config/ceilometer_agent_compute.json:3]: We can't connect to the libvirt socket and it doesn't matter if we use root or ceilometer user at the moment. Check https://issues.redhat.com/browse/OSPRH-3373 for more context. I haven't seen any additional issues when switching the user to ceilometer. When using the WA from the issue or switching SELinux to permissive, I can confirm that ceilometer has enough permissions.

**Comment by jlarriba** [comment]
/retest

There is a timeout connecting quay.io, it seems to be a temporary error, as it now works fine.

**Review by csibbitt — APPROVED** [review]

**Review by paramite — APPROVED** [review]

**Pull request merged**